resources/views/licensing.blade.php

```diff
@@ -64,13 +64,13 @@
             </table>
         </div>
 
-        <h6 class="mt-8">Core</h6>
+        <h6 class="mt-8">{{ __('Core') }}</h6>
         <div class="card p-0 mt-2">
             <table class="data-table">
                 <tr>
                     <td class="w-64 font-bold">
                         <span class="little-dot {{ $statamic->valid() ? 'bg-green-600' : 'bg-red-500' }} mr-2"></span>
-                        Statamic @if ($statamic->pro())<span class="text-pink">Pro</span>@else Free @endif
+                        Statamic @if ($statamic->pro())<span class="text-pink">{{ __('Pro') }}</span>@else {{ __('Free') }} @endif
```

Comment:
I don't know if we really want to change this. "Statamic Pro" is the name of the product, we don't change the name depending on the language.

Reply:
Alright. I suppose they (Pro, Free) is tips for users. If that is part of name so I will revert it.